Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.yml

```
@@ -0,0 +1,191 @@
+      type: string
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: number
+    - contextPath: WildFire.Report.Status
+      description: The status of the submissiom.
+      type: string
+    - contextPath: WildFire.Report.SHA256
+      description: SHA256 hash of the submission.
+      type: string
+    - contextPath: InfoFile.EntryID
+      description: The EntryID of the report file.
+      type: string
+    - contextPath: InfoFile.Extension
+      description: The extension of the report file.
+      type: string
+    - contextPath: InfoFile.Name
+      description: The name of the report file.
+      type: string
+    - contextPath: InfoFile.Info
+      description: Details of the report file.
+      type: string
+    - contextPath: InfoFile.Size
+      description: The size of the report file.
+      type: number
+    - contextPath: InfoFile.Type
+      description: The report file type.
+      type: string
+    - contextPath: WildFire.Report.Network.UDP.IP
+      description: Submission related IPs, in UDP protocol.
+      type: string
+    - contextPath: WildFire.Report.Network.UDP.Port
+      description: Submission related ports, in UDP protocol.
+      type: string
+    - contextPath: WildFire.Report.Network.TCP.IP
+      description: Submission related IPs, in TCP protocol.
+      type: string
+    - contextPath: WildFire.Report.Network.TCP.Port
+      description: Submission related ports, in TCP protocol.
+      type: string
+    - contextPath: WildFire.Report.Network.DNS.Query
+      description: Submission DNS queries.
+      type: string
+    - contextPath: WildFire.Report.Network.DNS.Response
+      description: Submission DNS responses.
+      type: string
+    - contextPath: WildFire.Report.Evidence.md5
+      description: Submission evidence MD5 hash.
+      type: string
+    - contextPath: WildFire.Report.Evidence.Text
+      description: Submission evidence text.
+      type: string
+    - contextPath: WildFire.Report.detection_reasons.description
+      description: Reason for the detection verdict.
+      type: string
+    - contextPath: WildFire.Report.detection_reasons.name
+      description: Name of the detection.
+      type: string
+    - contextPath: WildFire.Report.detection_reasons.type
+      description: Type of the detection.
+      type: string
+    - contextPath: WildFire.Report.detection_reasons.verdict
+      description: Verdict of the detection.
+      type: string
+    - contextPath: WildFire.Report.detection_reasons.artifacts
+      description: Artifacts of the detection reasons.
+      type: unknown
+    - contextPath: WildFire.Report.iocs
+      description: Associated IOCs.
+      type: unknown
+    - contextPath: WildFire.Report.verdict
+      description: The verdict of the report.
+      type: string
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.9.6.24067
+fromversion: 5.5.0
```

@ytauman needed from 6.5.0 only, correct?
```suggestion
fromversion: 6.5.0
```